lib/tilt/commonmarker.rb

```diff
@@ -49,7 +58,11 @@ def parse_options
 
     def render_options
       raw_options = RENDER_OPTIONS.select do |option|
-        options[option]
+        if INVERT_OPTIONS.include? option
+          !options[option]
+        else
+          options[option]
+        end
```

Comment:
`parse_options`/`render_options` setup code is similar enough and now complex enough to warrant extracting into a method.  However, that can be done later if you don't want to do it in this PR.